### PR TITLE
ClassroomRetrieveById with Tests Added

### PR DESCRIPTION
**Pull request by tajwal**
ClassroomRetrieveById

### CODE REVIEW

File: OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs [hunk 1]
```diff
@@ -67,5 +67,15 @@ public partial class ClassroomService : IClassroomService
 
 			return storageClassrooms;
 		});
+
+		public ValueTask<Classroom> RetrieveClassroomById(Guid classroomId) =>
+		TryCatch(async () =>
+		{
+			Classroom storageClassroom =
```
Comment (viralpandya) [inline OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs:74]: Validation tests and some validations are missing. Please check with other entities.

File: OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs [hunk 2]
```diff
@@ -67,5 +67,15 @@ public partial class ClassroomService : IClassroomService
...
+				await this.storageBroker.SelectClassroomByIdAsync(classroomId); await this.storageBroker.SelectClassroomByIdAsync(classroomId);
```
Comment (viralpandya) [inline OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs:75]: await this.storageBroker.SelectClassroomByIdAsync(classroomId);  is called twice here

File: OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs (hunk 1 above)
Comment (tajwal) [reply OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs:74]: I don't why it was coming two time as i did tested and debug many time, the duplicated lines are removed.

File: OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs (hunk 2 above)
Comment (tajwal) [reply OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs]: I think its was because of conflict, its removed now.

**Comment by hassanhabib** [comment]
Shutting this down for other similar work being completed.****

**Pull request closed without merging**